dataprofiler/data_readers/graph_data.py

```diff
@@ -12,9 +13,14 @@
 class GraphData(BaseData):
     """GraphData class to identify, read, and load graph data."""
 
-    data_type = "graph"
+    data_type: str = "graph"
 
-    def __init__(self, input_file_path=None, data=None, options=None):
+    def __init__(
+        self,
+        input_file_path: Optional[str] = None,
+        data: Optional[nx.Graph] = None,
```

Comment:
same comment from above in `csv_data` --> `:type data: multiple types` is type on `data` but we only allow `nx.graph` or `None` in this `__init__`. Should update so they match